### DISCUSSION TITLE
Does this Custom Firmware expand the available listening Frequencies?

### DISCUSSION THREAD

**Discussion by Kingharthacnut** [egzumer/uv-k5-firmware-custom#122] [Q&A] [answered]
Thanks for fixing the air band reception. (my radio arrives next week!)

**Comment by ErikS-web** [comment, marked as answer]
Answer is YES.
Original reception of 50-600 MHz is modified to 18-630 and 840-1300 MHz (630-840 not due to hardware limitation or internal IC).

Read our [Wike as a Manual](https://github.com/egzumer/uv-k5-firmware-custom/wiki).
Is located as Wiki at https://github.com/egzumer/uv-k5-firmware-custom/wiki.

Modified[ intro-page](https://github.com/egzumer/uv-k5-firmware-custom/wiki) with this info.
